templates/OpsWorksinVPC.template.erb

```diff
@@ -222,9 +209,13 @@
     "PrivateRoute" : {
       "Type" : "AWS::EC2::Route",
       "Properties" : {
-        "RouteTableId" : { "Ref" : "PrivateRouteTable" },
+        "RouteTableId" : {
```

Comment:
Here we're replacing the use of a NAT instance with the newish AWS NAT Gateway service. This was done per the example at https://aws.amazon.com/blogs/apn/taking-nat-to-the-next-level-in-aws-cloudformation-templates/